app-admin/yadm/yadm-3.2.2.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..10} )
+inherit bash-completion-r1 python-any-r1
+
+DESCRIPTION="Git based tool for managing dotfiles"
+HOMEPAGE="https://github.com/TheLocehiliosan/yadm"
+SRC_URI="https://github.com/TheLocehiliosan/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="test"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	app-shells/bash
+	app-crypt/gnupg
```

Comment:
Sort.

Reply:
i see pr was merged. i can open a revbump with sorted list, or i can add a note to myself to sort it on next version bump

Reply:
No need, 809e34279ce2af190a3b706ea2b9c4c77e72a7f2

Reply:
oh. awesome. thank you!

Reply:
No no, thank YOU! Especially if we can get python-3.11 support.